controllers/provider-openstack/example/30-worker.yaml

```
@@ -24,13 +24,22 @@ spec:
         apiVersion: openstack.provider.extensions.gardener.cloud/v1alpha1
         kind: CloudProfileConfig
         keystoneURL: https://localhost
+        machineImages:
+          - name: coreos
+            versions:
+              - version: 2135.6.0
+                image: openstackimagename
```

I don't understand this. What is `image` here if there is already an ID-per-region mapping? If it's optional I suggest 
```suggestion
              # image: openstackimagename
```
to make clear that either `image` or `regions[]` must be provided.